src/plugins/PasteHandler.ts

```diff
@@ -140,11 +140,14 @@ export default class PasteHandler extends Extension {
               const paste = this.editor.pasteParser.parse(
                 normalizePastedMarkdown(text)
               );
-              const slice = paste.slice(0);
 
-              const transaction = view.state.tr.replaceSelection(slice);
-              view.dispatch(transaction);
-              return true;
+              if ( paste ) {
```

Comment:
in which case it's falsy?

Reply:
this is just to satisfy typescript i don't know to be honest lol